pkg/cli/cc/cc.go

```diff
@@ -14,7 +16,16 @@ import (
 type CommonCtx struct {
 	Context context.Context
 	UI      *clui.UI
-	NoCheck bool
+	Config  *CLIConfig
+}
+
+type CLIConfig struct {
+	NoCheck           bool
+	DefaultConfigFile string
```

Comment:
I would rename this field to `TopazConfigFile`. "Default" makes it seem like there are other, non-default, files but a CLIConfig refers to exactly one topaz config file.